### DISCUSSION TITLE
How to hide a certain note in local graph?

### DISCUSSION THREAD

**Discussion by Eucliddd** [oleeskild/obsidian-digital-garden#255] [Q&A]
Like the native setting - "Excluded Files" - in Obsidian, how to make the notes hidden in Graph View?

**Comment by uroybd** [comment]
Add a `dg-hide-in-graph` to the frontmatter to hide in both local and global graphs.

**Reply by Eucliddd** [reply]
I have added `dg-hide-in-graph: true` in the frontmatter, but it didn't work for me.

**Reply by uroybd** [reply]
It didn't worked for me for the current release. However, compiling from current main branch worked. 

@oleeskild Can you please check if anything got missing in the release?

**Reply by oleeskild** [reply]
Right, I had completed the pull request, but forgot to pull the changes before creating a build for the release. 😅 I've released version 2.43.1 of the plugin. That one should work 🤞
